File: alpro-lioneljevon-9-10-2025/Versi C++/bilangangenap.cpp
#include <iostream>
#include <cstdio>
using namespace std;

int main() {
    cout << "\n=================================" << endl;
    cout << "\n";
    cout << "Mencari Bilangan Genap \n" << endl;

    int awal, akhir;
    cout << "Masukkan Angka awal : "; cin >> awal;
    cout << "Masukkan Angka akhir : "; cin >> akhir;

    cout << "\n=================================" << endl;
    printf("Bilangan Genap:\n");

    while(awal <= akhir) {
        awal += 1;
        if(awal % 2 == 0) {
            printf("%d\t|\n", awal);
        }
    }

    return 0;
}

File: alpro-lioneljevon-9-10-2025/bilangangenap.py
print("\n=================================")
print("\n")
print("Mencari Bilangan Genap \n")
awal = int(input("Masukkan Angka awal: "))
akhir_input = int(input("Masukkan Angka awal: "))

while awal <= akhir_input:
    awal += 1
    if awal%2 == 0:
        print("%d \t|" %(awal))


print("\n")
print("=======================\n")

print("Versi Metode For")
print("\n=================================")
print("\n")
print("Mencari Bilangan Genap \n")
awal = int(input("Masukkan Angka awal: "))
akhir_input = int(input("Masukkan Angka awal: "))

for awal in range(akhir_input+1):
    if awal % 2 == 0:
        print("%d \t|" % (awal))

print("\n")